banyan/resource_service_web.go

```diff
@@ -286,6 +286,11 @@ func WebSchema() (s map[string]*schema.Schema) {
 			Optional:    true,
 			Description: "access tier group which is associated with service",
 		},
+		"post_auth_redirect_path": {
```

Comment:
@rohitSangamnerkar default could be set to "/" or probably required to be set same as what we do from console.

Reply:
from console we set / as default value. 
Updated terraform config to set default value of post_auth_redirect_path.